vnext/Shared/Networking/WinRTWebSocketResource.cpp

```diff
@@ -334,6 +338,7 @@ IAsyncAction WinRTWebSocketResource2::SendPendingMessages() noexcept {
     auto result = async.ErrorCode();
     if (result < 0) {
       Fail(std::move(result), ErrorType::Send);
+      co_return;
```

Comment:
I wonder why is it needed.

Reply:
If we get a write error, we should stop processing further messages and notify the JavaScript layer.

See https://datatracker.ietf.org/doc/html/rfc6455#section-8.1